frontend/ui/src/components/ContactListItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from "react";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
+import ListItemAvatar from "@mui/material/ListItemAvatar";
+import Avatar from "@mui/material/Avatar";
+import { ContactCompanyList } from "./ContactCompanyList";
+import { ContactCheckbox } from "./ContactCheckbox";
+
+export const ContactListItem: FC<{
+  name: string;
```

Comment:
You could move name, profilePictureS3Id, companyName, logoS3Id, companyPosition, linkedInUrl to a Contact interfase and just import it. For example:

types.ts . Check if you can use defined Contact in `frontend/ui/src/types/UiContact.ts`
```
interface Contact {
    profilePictureS3Id: string;
   companyName: string;
   logoS3Id: string;
   companyPosition: string;
    linkedInURL: string;
}

```
ConstactListItem.tsx
```
import {Contact} from './types'
....
export const ContactListItem: FC<{contact: Contact, ...}> = (contact, index, checked, ...) 

```
ContactList.tsx (line 36)
```
<ContactListItem
              onChange={handleToggle}
              checked={checked}
              index={index}
             contact={contact}
            />

```